roles/sft-monitoring-certs/tasks/main.yml

```diff
@@ -1,53 +1,46 @@
-- name: Temporary file for the CA private key
+- name: Temporary directory for operations
   tempfile:
-  register: ca_private_key
+    state: directory
+  register: workspace
```

Comment:
Does it make sense to clean up the workspace at the end again? Or would it even break things?

Reply:
It wouldn't break things, but it is a tempdir, so I don't really see the point of doing it.

Reply:
fair point.